doctest/parts/doctest.cpp

```diff
@@ -3076,9 +3077,15 @@ namespace {
             }
         }
 
-        void test_run_start() override { printIntro(); }
+        void test_run_start() override {
+            if(!opt.minimal)
+                printIntro();
+        }
 
         void test_run_end(const TestRunStats& p) override {
+            if(opt.minimal)
+                return;
```

Comment:
I think it makes sense to print the outro if anything has failed - as a summary. There's a variable a few lines below called `anythingFailed` which could be checked for that

Reply:
I'm wondering if this should only depend on the failure of test cases, since assertions could fail with `doctest::{may, should}_fail` and perhaps even `doctest::no_output` in which case I wouldn't expect a summary with minimal.

If you disagree I'll keep it as is (and fix the test).

Reply:
You are right, I hadn't considered those cases!